lib/basechurch/engine.rb

```diff
@@ -5,6 +5,7 @@
 require 'friendly_id'
 require 'validate_url'
 require 'rails-api'
+require 'has_attachment'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.